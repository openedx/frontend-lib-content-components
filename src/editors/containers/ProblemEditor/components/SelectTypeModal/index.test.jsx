import React from 'react';
import { shallow } from '@edx/react-unit-test-utils';
import * as module from '.';

jest.mock('./hooks', () => ({
  selectHooks: jest.fn(() => ({
    selected: 'mOcKsELEcted',
    setSelected: jest.fn().mockName('setSelected'),
  })),
  useArrowNav: jest.fn().mockName('useArrowNav'),
}));

describe('SelectTypeModal', () => {
  const props = {
    onClose: jest.fn(),
  };

  test('snapshot', () => {
    expect(shallow(<module.SelectTypeModal {...props} />).snapshot).toMatchSnapshot();
  });
});
