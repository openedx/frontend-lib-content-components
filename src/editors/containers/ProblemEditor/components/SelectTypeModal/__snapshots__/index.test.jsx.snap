// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SelectTypeModal snapshot 1`] = `
<injectIntl(ShimmedIntlComponent)
  onClose={[MockFunction]}
  selected="mOcKsELEcted"
>
  <Row
    className="justify-content-center"
  >
    <Stack
      className="flex-wrap mb-6"
      direction="horizontal"
      gap={4}
    >
      <injectIntl(ShimmedIntlComponent)
        selected="mOcKsELEcted"
        setSelected={[MockFunction setSelected]}
      />
      <injectIntl(ShimmedIntlComponent)
        problemType="mOcKsELEcted"
      />
    </Stack>
  </Row>
</injectIntl(ShimmedIntlComponent)>
`;
